content/hardware/05.pro-solutions/solutions-and-kits/opta/tutorials/getting-started-connectivity/content.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 'Using BLE, WiFi and Ethernet on the Arduino Opta'
+description: "Learn how to make use of the Arduino Opta's connectivity features"
+difficulty: beginner
+tags:
+  - Wi-Fi®
+  - Ethernet
+  - Bluetooth®
+  - Connectivity
+author: 'Benjamin Dannegård'
+libraries:
+  - name: Arduino BLE
+    url: https://www.arduino.cc/reference/en/libraries/arduinoble/
+software:
+  - ide-v1
+  - ide-v2
+  - web-editor
+hardware:
+  - hardware/05.pro-solutions/solutions-and-kits/opta
+---
+
+## Overview
+
+The Arduino® Opta is a powerful PLC device that has many features, allowing you to customize its use for your solution. Among these features are the standard connectivity features: Wi-Fi®, Ethernet and Bluetooth®. In this tutorial we will go through how to use these features with the Arduino IDE and the Arduino Opta.
```

Comment:
```suggestion
The Arduino Opta® is a powerful PLC device that has many features, allowing you to customize its use for your solution. Among these features are the standard connectivity features: Wi-Fi®, Ethernet and Bluetooth®. In this tutorial we will go through how to use these features with the Arduino IDE and the Arduino Opta®.
```
Trademark addition for Arduino Opta® (Recent addition)

Reply:
idem